spec/models/course/lesson_plan_item_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Course::LessonPlanItem, type: :model do
+  it { is_expected.to validate_numericality_of(:base_exp).only_integer }
+  it { is_expected.to validate_numericality_of(:time_bonus_exp).only_integer }
+  it { is_expected.to validate_numericality_of(:extra_bonus_exp).only_integer }
+
+  let!(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    let(:lesson_plan_item) { FactoryGirl.create :course_lesson_plan_item }
+    context '.total_exp' do
```

Comment:
should be <code>describe '#total_exp' </code>

And can you also add a test case for <code>acts_as_lesson_plan_item</code> ?